src/confd/yang/infix-system.yang

```diff
@@ -256,6 +301,37 @@ module infix-system {
     }
   }
 
+  augment "/sys:system-state" {
+    container ntp {
+      description "NTP status";
+      container sources {
+        list source {
+          key address;
+          leaf address {
+            type inet:ip-address;
+            description "Address to connected NTP server";
```

Comment:
Iirc, it's UDP, so maybe replace/drop "to connected" ... perhaps just "Server address".